docs/Configuration.md

```diff
@@ -456,8 +456,12 @@ _Note: Any global variables that are defined through `globalSetup` can only be r
 
 _Note: While code transformation is applied to the linked setup-file, Jest will **not** transform any code in `node_modules`. This is due to the need to load the actual transformers (e.g. `babel` or `typescript`) to perform transformation._
 
+\_Note: User can access Jest [global config]()https://github.com/facebook/jest/blob/main/packages/jest-types/src/Config.ts#L282 and [project config](https://github.com/facebook/jest/blob/main/packages/jest-types/src/Config.ts#L347) as arguments for the function.
```

Comment:
doc changes also need to be added [here](https://github.com/facebook/jest/blob/main/website/versioned_docs/version-27.5/Configuration.md) I think....

Reply:
If it is in **27.5**, it should be modified there. But we are not sure yet if it is in **27.5**, I don't think that should be added.